packages/terra-hookshot/themeable-variables.json

```diff
@@ -0,0 +1 @@
pupContent":{"--terra-popup-content-header-border-bottom":"1px solid #dedfe0","--terra-popup-content-close-background":"0 0","--terra-popup-content-close-border":"0","--terra-popup-content-close-color":"#111","--terra-popup-content-close-font-size":"1.45rem","--terra-popup-content-close-font-weight":"bold","--terra-popup-content-close-line-height":"0","--terra-popup-content-close-margin-left":"0.3571rem","--terra-popup-content-close-margin-right":"0","--terra-popup-content-close-padding":"4px","--terra-popup-content-close-hover-color":"#999","--terra-popup-content-inner-background-color":"#fff","--terra-popup-content-inner-border":"1px solid #b2b5b6","--terra-popup-content-inner-box-shadow, 0 4px 8px 0 rgba(0, 0, 0, 0.35), 0 4px 3px -3px rgba(0, 0, 0":"0.5)","--terra-popup-content-margin":"10px"},"PopupOverlay":{"--terra-popup-overlay-background-color":"#fff","--terra-popup-overlay-opacity":"0.01","--terra-popup-overlay-z-index":"7000"},"ProgressBar":{"--terra-progress-bar-height-small":"0.8rem","--terra-progress-bar-height-tiny":"0.5rem","--terra-progress-bar-height-medium":"1rem","--terra-progress-bar-height-large":"1.4rem","--terra-progress-bar-height-huge":"1.7rem"},"SlidePanel":{"--terra-slide-panel-panel-width-small":"320px","--terra-slide-panel-panel-width-large":"60%"},"Table":{"--terra-table-background-color":"#fff","--terra-table-border":"0","--terra-table-cell-border":"0","--terra-table-row-border-vertical":"1px solid #c8cacb","--terra-table-thead-background-color":"#dedfe0","--terra-table-thead-tr-border-bottom":"0","--terra-table-thead-tr-border-top":"1px solid #c8cacb","--terra-table-thead-th-color":"#1c1f21","--terra-table-thead-th-font-size":"1rem","--terra-table-thead-th-font-weight":"bold","--terra-table-row-border-horizontal":"1px solid #c8cacb","--terra-table-tfoot-tr-border-bottom":"1px solid #c8cacb","--terra-table-tfoot-tr-border-top":"0","--terra-table-subheader-background-color":"#c8cacb","--terra-table-subheader-color":"#1c1f21","--terra-table-subheader-font-size":"1rem","--terra-table-subheader-font-weight":"bold","--terra-table-sort-indicator-color":"#1c1f21","--terra-table-cell-min-width-tiny":"3rem","--terra-table-cell-min-width-small":"4.25rem","--terra-table-cell-min-width-medium":"5.75rem","--terra-table-cell-min-width-large":"8.5rem","--terra-table-cell-min-width-huge":"10.5rem","--terra-table-cell-padding-bottom":"0.5em","--terra-table-cell-padding-left":"0.75em","--terra-table-cell-padding-right":"0.75em","--terra-table-cell-padding-top":"0.5em","--terra-table-row-striped-background-color":"#f4f4f4","--terra-table-row-striped-selected-background-color":"#d4ecfb","--terra-table-row-striped-hover-background-color":"#e2f2fc","--terra-table-row-striped-selected-hover-background-color":"#bde1f8","--terra-table-row-selected-background-color":"#ebf6fd","--terra-table-row-selected-border-color":"#0092e0","--terra-table-row-hover-background-color":"#ebf6fd","--terra-table-row-selected-hover-background-color":"#d4ecfb"}}
```

Comment:
doesn't the new version that generates this string create a more readable output?

Reply:
I think it should, but for some reason it isn't. Anyone have any thoughts?

Reply:
Its odd to me this file is even being generated in this directory. I thought @JakeLaCombe had an update to only generate this in the terra-site directory.

Reply:
We should hard code the [file path](https://github.com/cerner/terra-core/blob/master/packages/terra-site/theming-plugin.js#L36) the theming file is output to to always output in terra-site instead of rely on the `__dir` variable.